### PR TITLE
Add wallet list to the ink docs

### PR DESCRIPTION
**Pull request by saltict**
## Description
SubWallet, Polkadot{.js}, and Talisman are the walllets that support interacting with Ink smart contracts.
SubWallet is the first wallet to support PSP22 and PSP34 tokens written with Ink smart contracts. SubWallet's compatibility enhances the accessibility of these tokens, allowing users to seamlessly manage and interact with them. This pull request aims to enhance the Ink documentation by adding SubWallet as a supported wallet for interacting with Ink smart contracts.

## Details
1. Modified `intro.mdx`: Added SubWallet to the list of supported wallets in the Ink documentation and linked to `https://subwallet.app`.
2. Added `/static/img/wallets/subwallet.svg`, `/static/img/wallets/polkadot-js.svg`, `/static/img/wallets/talisman.svg`.
3. Modified `custom.css` for the wallet section.

## Screenshot
image
image

These screenshots provides a visual representation of the changes that have been made.

## Testing
I deployed the changes to the [demo](https://sw-ink-docs.netlify.app) for preview purposes.

## Additional Notes
SubWallet has already added support for Ink:
  - SubWallet Issue:
    - [GitHub Issue](https://github.com/Koniverse/SubWallet-Extension/issues/477)
  - Twitter posts about this:
    - [Polkadot Tweet](https://twitter.com/Polkadot/status/1597221322265833472)
    - [ParasHQ Tweet](https://twitter.com/ParasHQ/status/1663956656563310592?s=20)
    - [AstarNetwork Tweet](https://twitter.com/AstarNetwork/status/1666438486591938560)

## Contributors
- @saltict

Your consideration of this pull request is highly appreciated. Thank you!

Sincerely,
Peter Mai

### CODE REVIEW

**Comment by SkymanOne** [comment]
Would be nice if you added a quick article in `Third Party Tools & Libraries` about the supported PSP standards by Subwallet and how they are displayed

File: docs/intro/intro.mdx [hunk 1]
```diff
@@ -198,6 +198,16 @@ hide_table_of_contents: false
     <div className="col">&nbsp;</div>
 </div>
 
+## What wallet can I use?
```
Comment (SkymanOne) [inline docs/intro/intro.mdx:201]:
```suggestion
## What wallets can I use?
```
I hope there will be more :)

**Review by SkymanOne — APPROVED** [review]

File: docs/intro/intro.mdx (hunk 1 above)
Comment (cmichi) [reply docs/intro/intro.mdx]: Plus one for this suggestion.

**Review by cmichi — CHANGES_REQUESTED** [review]
Thanks for the PR!

So strictly speaking you can also use e.g. `polkadot-js` or Talisman as wallets to interact with contracts. So the title "Which wallet can I use?" can appear as frightening if we only refer SubWallet here.

It's clear to me that you mean it in the context of PSP standards supported by wallets. I think it would be best to under the title "Which wallet can I use?" have a bunch of logos and then a table that compares the wallets against each other.

So something like

|   |  SubWallet  | `polkadot-js`  | Talisman  |
|---|---|---|---|
| Interacting with Contracts  | ✅  | ✅  | ✅   |
| NFT's (PSP-34)  |  ✅ | ﹣  | ﹣  |

Please also move the "Which wallets can I use?" section on the intro page below the examples.

PS: Kudos for putting the graphic together like the others! We did it this way so that it also works when the light theme is switched on.

**Comment by saltict** [comment]
Hi @SkymanOne and @cmichi, thank you for your suggestions.

I have updated the PR. Please help review and approve it if everything is okay

**Review by cmichi — APPROVED** [review]
Thank you a lot for this contribution!

I'm sorry for the delay in response, there were vacations and we're in the process of catching up.

**Comment by cmichi** [comment]
@saltict Can I ask you to merge `master`? There are some conflicts, they shouldn't be hard to resolve.

**Comment by saltict** [comment]
> @saltict Can I ask you to merge `master`? There are some conflicts, they shouldn't be hard to resolve.

I resolved the problems by backing up the old master branch, sync-fork, and merging it into new master branch.

**Pull request merged**